stages/apps/lead/nginx.tf

```diff
@@ -8,3 +8,13 @@ module "app_nginx" {
   issuer_kind    = module.cluster_issuer.issuer_kind
   namespace      = module.toolchain_namespace.name
 }
+
+module "monitoring_nginx" {
```

Comment:
what is this needed for?

Reply:
We do not need that, forgot to take that out.